### ISSUE TITLE
ECDSA alg Fatal error

### ISSUE THREAD

**Issue by paceto256** [namshi/jose#98]
Fatal error: Class 'phpseclib\File\ASN1' not found in /home/paceto/proj/sgapi/vendor/namshi/jose/src/Namshi/JOSE/Signer/OpenSSL/ECDSA.php on line 46

ref to old issue:
https://github.com/namshi/jose/issues/55

**Comment by odino** [comment]
hi @paceto256 would you mind sending a PR?

**Comment by paceto256** [comment]
to put "phpseclib/phpseclib": "^2.0" in "require" block ?

**Comment by paceto256** [comment]
Not sure what you mean by PR but this is my composer row
"require": {
    "namshi/jose": "^7.1"
}

Fixed the issue by adding "phpseclib/phpseclib": "^2.0" to require.

    "require": {
        "namshi/jose": "^7.1",
        "phpseclib/phpseclib": "^2.0"
    }


It's needed since 
Namshi\JOSE\Signer\OpenSSL\ECDSA depends on phpseclib\File\ASN1

**Comment by odino** [comment]
yes, I mean a pull request :) We made phpseclib an optional dependency so if you want to use ECDSA signers you should include it separately
